superset-frontend/src/explore/components/QueryAndSaveBtns.jsx

```diff
@@ -115,6 +115,7 @@ export default function QueryAndSaveBtns({
             data-toggle="modal"
             disabled={saveButtonDisabled}
             onClick={onSave}
+            data-test="save-button"
```

Comment:
Wondering if something more-explicit, e.g. `explore-save-button` might prevent a future issue where we have two `save-button` elements, like this one and some other Save button in a Popover or Modal.